modulefiles/GDAS/hera.intel.lua

```diff
@@ -83,9 +83,9 @@ local mpinproc = '-n'
 setenv('MPIEXEC_EXEC', mpiexec)
 setenv('MPIEXEC_NPROC', mpinproc)
 
-setenv("CRTM_FIX","/scratch1/NCEPDEV/da/Cory.R.Martin/GDASApp/fix/crtm/2.4.0")
-setenv("GDASAPP_TESTDATA","/scratch1/NCEPDEV/da/Cory.R.Martin/CI/GDASApp/data")
-setenv("GDASAPP_UNIT_TEST_DATA_PATH", "/scratch1/NCEPDEV/da/Cory.R.Martin/CI/GDASApp/data/test")
+setenv("CRTM_FIX","/scratch1/NCEPDEV/da/role.jedipara/CI/GDASApp/fix/crtm/2.4.0")
```

Comment:
since we are making these changes now, I'm wondering if it makes more sense to make `GDASApp` the top level directory, then have `fix` and `testdata` and `unittestdata` or something like that. Meaning I guess maybe now is the time to figure out the "best" structure for going forward

Reply:
Sure, now is the time to make the switch to whatever we feel is better.  I can refactor to
- `GDASApp`
   - `fix`
   - `testdata`
   - `unittestdata`

Reply:
that's just my first suggestion, not sure which is the best. Do you have any preference @RussTreadon-NOAA on what makes the most sense logically? Putting on a more generic hat, if we also host regional/hurricane files, do we want it to be `fix/GDAS` or `GDAS/fix`, etc.?

Reply:
We have [RDASApp](https://github.com/NOAA-EMC/RDASApp) and [HDASApp](https://github.com/NOAA-EMC/HDASApp).  Given this , I don't think GDASApp needs to host regional or hurricane files.  

For JEDI fix files I think DASApp specific directories are preferable (e.g., `GDASApp/fix`).  NCO will want a clean GDASApp, RDASApp, HDASApp fix directory without fix files from other *DASApp. 

When we move to `crtm/3` we should be able to remove `CRTM_FIX` from the GDASApp modulesfiles.  `crtm/3` uses netcdf coefficient files so we don't need to worry about different endianness for GSI and JEDI crtm coefficients.